backend/database.js

```diff
@@ -92,13 +93,119 @@ async function getUserID(email) {
     })
 }
 
+async function getAllTicketsFromUser(userID) {
+    const query = 'SELECT * FROM Ticket WHERE fk_userID = ?';
+    return new Promise((resolve, reject) => {
+        db.all(query, [userID], (err, row) => {
+            if (err) {
+                reject(err);
+            } else {
+                resolve(row);
+            }
+        })
+    })
+}
+
+async function getLastVariant(userID, Date) {
+    const query = 'SELECT * FROM Ticket WHERE fk_userID = ? AND Date = ?'
```

Comment:
U use a lot of 'SELECT * FROM'. You clould maby create a constant like: 
SELECT_FROM if you want